tests/command_line/test_cluster_unit_cell.py

```diff
@@ -27,6 +28,77 @@ def test_dials_cluster_unit_cell_command_line(dials_regression, run_in_tmpdir):
     assert os.path.exists("cluster_unit_cell.png")
 
 
+def test_dials_cluster_unit_cell_command_line_output_files(dials_regression, tmp_path):
+    pytest.importorskip("scipy")
+    pytest.importorskip("xfel")
+
+    data_dir = os.path.join(
+        dials_regression, "refinement_test_data", "multi_narrow_wedges"
+    )
+    experiments = glob.glob(os.path.join(data_dir, "data/sweep_*/experiments.json"))
+    reflections = glob.glob(os.path.join(data_dir, "data/sweep_*/indexed.pickle"))
+
+    # first combine experiments
+    result = procrunner.run(
+        command=["dials.combine_experiments"] + experiments + reflections,
+        working_directory=tmp_path,
+    )
+    assert not result.returncode
+    assert (tmp_path / "combined.refl").is_file()
+    assert (tmp_path / "combined.expt").is_file()
+
+    result = procrunner.run(
+        command=[
+            "dials.cluster_unit_cell",
+            "plot.show=False",
+            tmp_path / "combined.refl",
+            tmp_path / "combined.expt",
+            "output.clusters=True",
+            "threshold=40",
+        ],
+        working_directory=tmp_path,
+    )
+    assert not result.returncode
+    assert (tmp_path / "cluster_unit_cell.png").is_file()
+    assert (tmp_path / "cluster_0.refl").is_file()
+    assert (tmp_path / "cluster_0.expt").is_file()
+    expts = load.experiment_list(tmp_path / "cluster_0.expt", check_format=False)
+    assert len(expts) == 101
+    assert (tmp_path / "cluster_1.refl").is_file()
+    assert (tmp_path / "cluster_1.expt").is_file()
+    expts = load.experiment_list(tmp_path / "cluster_1.expt", check_format=False)
+    assert len(expts) == 1
+    assert (tmp_path / "cluster_2.refl").is_file()
+    assert (tmp_path / "cluster_2.expt").is_file()
+    expts = load.experiment_list(tmp_path / "cluster_2.expt", check_format=False)
+    assert len(expts) == 1
+
+    result = procrunner.run(
+        command=[
+            "dials.split_experiments",
+            tmp_path / "combined.refl",
+            tmp_path / "combined.expt",
+        ],
+        working_directory=tmp_path,
+    )
+    assert not result.returncode
+    experiments = glob.glob(os.path.join(tmp_path, "split_*.expt"))
+    reflections = glob.glob(os.path.join(tmp_path, "split_*.refl"))
```

Comment:
I think the following should work (and be much simpler):
```suggestion
    experiments = tmp_path.glob("split_*.expt")
    reflections = tmp_path.glob("split_*.refl")
```